Shared/Samples/Set basemap/SetBasemapView.swift

```diff
@@ -32,19 +32,24 @@ struct SetBasemapView: View {
     }()
     
     var body: some View {
-        MapView(map: map)
-            .overlay(alignment: .topTrailing) {
-                if isShowingBasemapGallery {
-                    BasemapGallery(geoModel: map)
-                        .style(.automatic())
+        GeometryReader { geometryProxy in
+            MapView(map: map)
+                .overlay(alignment: .topTrailing) {
+                    if isShowingBasemapGallery {
+                        BasemapGallery(geoModel: map)
+                            .style(.automatic())
+                            .frame(maxWidth: geometryProxy.size.width / 3)
+                            .esriBorder()
```

Comment:
In #110 this was removed. Can you add a comment here on why it is used in the new version? So we can refer to here when we review the code in the future.

Reply:
In 110, it was removed because the component was adding its own border. That's no longer true, its up to a user of the component to supply their own border now.